flink-formats/flink-parquet/pom.xml

```diff
@@ -141,6 +158,14 @@ under the License.
 	</dependencies>
 
 	<build>
+		<extensions>
```

Comment:
This is used to detect the os version (like linux, mac, window...)

As a whole, to test the protobuf-based writer, we have add a .proto file and wants to generate the corresponding java codes from the file. protoc compiler is used to generate it. However, protoc is provided to maven as pre-compiled binary executions and different version is required for different OS. therefore

1. `kr.motd.maven:os-maven-plugin` is used to detect the OS version, it will create a variable called ${os.detected.classifier}
2. `org.xolstice.maven.plugins:protobuf-maven-plugin` downloads the protoc according to ${os.detected.classifier} and do the generation.